### PR TITLE
spejstore - first try

### PR DESCRIPTION
**Pull request by wprzyb**

### CODE REVIEW

File: spejstore/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    restart: always
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
+    secrets:
+      - spejstore_clientid
+      - spejstore_secret
+    environment:
+      SPEJSTORE_CLIENT_ID: $$(cat /run/secrets/spejstore_clientid)
+      SPEJSTORE_SECRET: $$(cat /run/secrets/spejstore_secret)
+    volumes:
+      - .:/code
```
Comment (hbrylkowski) [inline spejstore/docker-compose.yml:22]: That volume rather has no sense

File: spejstore/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+version: "3"
+services:
+  db:
+    image: ghcr.io/hackerspacekrk/spejstore/postgres-hstore:latest
+    build: postgres-hstore
```
Comment (hbrylkowski) [inline spejstore/docker-compose.yml:5]: probably leftover from copying

File: spejstore/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      - "traefik.http.routers.spejstore.rule=Host(`spejstore.apps.hskrk.pl`)"
+      - "traefik.http.routers.spejstore.middlewares=authentik@docker"
+      - "traefik.http.routers.spejstore.tls=true"
+      - "traefik.http.routers.spejstore.tls.certresolver=myresolver"
+
+volumes:
+  spejstore-pg:
+
+secrets:
+  spejstore_secret:
```
Comment (hbrylkowski) [inline spejstore/docker-compose.yml:42]: just be sure to add those secrets to swarm *BEFORE* you start everything

**Review by hbrylkowski — CHANGES_REQUESTED** [review]

File: spejstore/docker-compose.yml (hunk 2 above)
Comment (wprzyb) [reply spejstore/docker-compose.yml:5]: Oh, cheeky lil bastard

File: spejstore/docker-compose.yml (hunk 1 above)
Comment (wprzyb) [reply spejstore/docker-compose.yml:22]: Yeah, need to figure out if it's even needed

File: spejstore/docker-compose.yml (hunk 3 above)
Comment (wprzyb) [reply spejstore/docker-compose.yml:42]: To swarm = to portainer?

File: spejstore/docker-compose.yml (hunk 1 above)
Comment (wprzyb) [reply spejstore/docker-compose.yml]: let's try and assume it isn't

File: spejstore/docker-compose.yml (hunk 2 above)
Comment (wprzyb) [reply spejstore/docker-compose.yml]: fixd

File: spejstore/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    ports:
+      - "8021:8000"
+    depends_on:
+      - db
+    env_file:
+      - spejstore.env
+    labels:
+      - "traefik.enable=true"
+      - "traefik.docker.network=system_default"
+      - "traefik.http.services.spejstore.loadbalancer.server.port=1880"
```
Comment (zagura) [inline spejstore/docker-compose.yml:29]: Which port number is used? 8000/8021 or 1880?

Comment (wprzyb) [reply spejstore/docker-compose.yml:29]: Good one, 8021 (as SSO was on 8037)

**Review by zagura — APPROVED** [review]

**Review by hbrylkowski — APPROVED** [review]

**Pull request merged**